js/components/Editor.css

```diff
@@ -4,8 +4,16 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+/**
+ * There's a known codemirror flex issue with chrome that this addresses.
+ * BUG https://github.com/jlongster/debugger.html/issues/63
+ */
 .editor {
-  flex: 1;
+  position: absolute;
+  height: calc(100% - 30px);
```

Comment:
Yeah, that makes me really sad. Flexbox was supposed to allow us to avoid this kind of stuff. But I guess it works for now.